commands/files/serialfile.go

```diff
@@ -84,10 +84,25 @@ func (f *serialFile) NextFile() (File, error) {
 	// open the next file
 	fileName := fp.Join(f.name, stat.Name())
 	filePath := fp.Join(f.path, stat.Name())
+	st, err := os.Lstat(filePath)
```

Comment:
The default should be dereference (current) or don't dereference for all symlinks.
Why should `ipfs add aSymlink` be dereferenced but `ipfs add aSymlink aFile` not?